Gifski/AppDelegate.swift

```diff
@@ -63,4 +64,14 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		Crashlytics.recordNonFatalError(error: error)
 		return error
 	}
+
+	/// This is called from NSApp as a service resolver
+	@objc
+	func convertToGif(_ pasteboard: NSPasteboard, userData: String, error: NSErrorPointer) {
```

Comment:
Can you move this into an extension? Maybe we should also call it `convertToGifService`?

Reply:
Sure, will move to extension. The name would be quite confusing for me, as this would suggest we would have some "gif service". But I see where you're coming from, you just want to indicate, in the function name, that this is called from a service right? I think we could leave the name as is with the comment and extension for now, but again up to you.

Reply:
Alright, we don't have to change the name. It just bothers me in general how implicit all these old Cocoa APIs are.